libs/common/include/launchdarkly/attribute_reference.hpp

```diff
@@ -15,8 +15,9 @@ namespace launchdarkly {
  * launchdarkly::Context::Get, or to identify an attribute or nested value that
  * should be considered private
  * with launchdarkly::AttributesBuilder<BuilderReturn, BuildType>::SetPrivate or
- * launchdarkly::AttributesBuilder<BuilderReturn, BuildType>::AddPrivateAttribute
- * (the SDK configuration can also have a list of private attribute references).
+ * launchdarkly::AttributesBuilder<BuilderReturn,
```

Comment:
I think that the auto formatter bungled this, and that it may cause problems with the docs.

Reply:
Haha, it was one character too long - I removed the gratuitous space for the template params and fixed it that way.